src/App.tsx

```diff
@@ -11,12 +11,15 @@ import TableRow from '@mui/material/TableRow';
 import Paper from '@mui/material/Paper';
 import { TableHead } from "@mui/material";
 
+type getMessageReturn = [string[],string[],string[],string[],number[]]
```

Comment:
This is a very good thing to highlight for #19 this should be a class because it's much more readable.